gammapy/datasets/make.py

```diff
@@ -64,3 +72,158 @@ def sigma_energy_theta(energy, theta, sigma):
                                        zenith=zenith_hi)
 
     return psf
+
+
+def make_test_observation_table(observatory_name, n_obs, debug=False):
+    """Make a test observation table.
+
+    For the moment, only random observation tables are created.
+
+    Parameters
+    ----------
+    observatory_name : string
+    	name of the observatory; a list of choices is given in `~gammapy.obs.observatory_locations`
+    n_obs : integer
+    	number of observations for the obs table
+    debug : bool
+    	show UTC times instead of seconds after the reference
+
+    Returns
+    -------
+    obs_table : `~gammapy.obs.ObservationTable`
+    	observation table
+    """
+    n_obs_start = 1
+
+    obs_table = ObservationTable()
+
+    # build a time reference as the start of 2010
+    dateref = Time('2010-01-01 00:00:00', format='iso', scale='utc')
+    # TODO: using format "iso" for `~astropy.Time` for now; eventually change it
+    # to "fits" after the next astropy stable release (>1.0) is out.
+    dateref_mjd_fra, dateref_mjd_int = np.modf(dateref.mjd)
+
+    # header
+    header = {'OBSERVATORY_NAME': observatory_name,
+              'MJDREFI': dateref_mjd_int, 'MJDREFF': dateref_mjd_fra}
+    ObservationTable.meta = header
+
+    # obs id
+    obs_id = np.arange(n_obs_start, n_obs_start + n_obs)
+    col_obs_id = Column(name='OBS_ID', data=obs_id)
+    obs_table.add_column(col_obs_id)
+
+    # TODO: `~astropy.table.Table` doesn't handle `~astropy.time.TimeDelta` columns
```

Comment:
I don't see any reason why we should use `TimeDelta`. Why not simply use `Quantity` and remove this TODO?